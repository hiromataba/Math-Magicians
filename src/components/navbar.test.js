import React from 'react';
import renderer from 'react-test-renderer';
import Navbar from './navbar';

it('expect quote to match snapshot', () => {
  const tree = renderer.create(<Navbar />).toJSON();
  expect(tree).toMatchSnapshot();
});
