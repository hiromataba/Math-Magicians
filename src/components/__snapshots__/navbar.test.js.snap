// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect quote to match snapshot 1`] = `
<div
  className="navbar"
>
  <div
    className="logo"
  >
    <a
      href="/"
      style={
        Object {
          "color": "#f28766",
        }
      }
    >
      <h3>
        Math Magician
      </h3>
    </a>
  </div>
  <ul
    className="nav-links"
  >
    <li>
      <a
        href="/"
      >
        Home
      </a>
    </li>
    <li>
      <a
        href="/calculator"
      >
        Calculator
      </a>
    </li>
    <li>
      <a
        href="/quote"
      >
        Quotes
      </a>
    </li>
  </ul>
</div>
`;
